source/stylesheets/modules/_toc.scss

```diff
@@ -44,3 +44,7 @@
     display: none;
   }
 }
+
+.toc-link--in-view {
```

Comment:
Feels weird to have a modifier without having declared the element itself? Might make sense to have a `toc__link` class to add to the links in the ToC, but we'd need to modify the ToC generator to add the class, and that's a lot of extra HTML to send over the wire. Maybe not so  bad once gzipped though?